src/editor/index.js

```diff
@@ -40,8 +45,9 @@ if ( isListing() ) {
 			icon: null,
 		} );
 
-		if ( selfServeEnabled ) {
-			// Register featured listing sidebar.
+		// Register featured listing sidebar.
+		// We don't want to show the Featured Listing UI to customers, otherwise anyone could just make their listings featured.
```

Comment:
Does this mean that there's no backend check for this?

Reply:
This line now also checks `isListingCustomer`, which is set on the back-end via https://github.com/Automattic/newspack-listings/pull/135/files#diff-4da2fe15473cc4696a1b3b1b06b348e753e63151ac5b0ac727399924e99794b3R1386-R1400 and localized so we can use it here.